docs/dev/table_api.md

```diff
@@ -1485,14 +1485,14 @@ Joins, set operations, and non-windowed aggregations are not supported yet.
 
 {% top %}
 
-### Group Windows
+### Windows
```

Comment:
I'd like called them `Windows`. All of the names are `Tumble`,`Slide`,`Session` and `Over`. It's consistent with the interface of the table API. Because we only have`window(...)` method in `Table.scala`. although the param of  `window(overWindows: OverWindow*)` and `window(window: Window)` a little difference. 
 When user using the tableAPI/SQL they only notice that  there are 4 kinds of windows in FLINK.
They are  `Tumble`, `Slide/Hup`, `Session` and `Over`. We'd better not reflect the concept of group window. What do you think?

Thanks,
SunJincheng